script/Ciel/Day 15/15_01A_CIEL15A_1.txt

```diff
@@ -750,18 +773,18 @@ And she hasn't developed a relationship with you at all, either..."
 [sha:f86a236205271986dd074ce74bd0ab892c0ca5fb]{
 -- Page 45, Offset 41518.
 -- 「ねえ、なんでそういうコトだけ明確に分かるのおまえ？」
-"Hey, why is %{i}that%{/i} the only thing you're quick on the uptake about?"
+"Hey, why is %{i}that%{/i} the one thing you actually %{i}can%{/i} tell right away?"
```

Comment:
Double italics feels a bit excessive to me.
> ... the one thing you can tell right away?
> ... the one thing you can tell for sure?

Reply:
Then I'd rather remove the first italic